src/sprite_actor.cpp

```diff
@@ -267,7 +264,7 @@ void Sprite_Actor::OnBattlercharsetReady(FileRequestResult* result, int32_t batt
 
 void Sprite_Actor::Draw(Bitmap& dst) {
 	auto* battler = GetBattler();
-	if (battler->IsHidden()) {
+	if (battler->IsHidden() || do_not_draw) {
```

Comment:
Is this fixing some other bug? It would be good to describe all the things implemented and bugs fixed in the commit message.

Reply:
Yes, this one is the fix for #1708. The `do_not_draw` variable is set to true if the CBA battler name is empty. I'm going to update the commit message accordingly when the commit gets changed.

Reply:
can you add a comment in the code?

Reply:
Added the comment now.